lmdeploy/lite/quantization/awq.py

```diff
@@ -116,11 +118,15 @@ def skipped_module(name: str):
 
 
 @torch.no_grad()
-def get_weight_scale(weight, q_group_size=-1):
+def get_weight_scale(weight, q_group_size=-1, clamp_zeros: bool = False):
     org_shape = weight.shape
     if q_group_size > 0:
         weight = weight.view(-1, q_group_size)
-    scale = weight.abs() / weight.abs().amax(dim=1, keepdim=True)
+    if clamp_zeros:
+        scale = weight.abs() / weight.abs().amax(dim=1,
+                                                 keepdim=True).clamp(min=1e-4)
+    else:
+        scale = weight.abs() / weight.abs().amax(dim=1, keepdim=True)
```

Comment:
Can we check if `weight.abs().amax(dim=1, keepdim=True)` equals 0?
If it is zero, can we clamp zero to a small value by default? If we can, may provide warning to users, informing it might cause performance degradation.

Reply:
My another concern is the justifiability of `--clamp-zero` option
Because it is clear that the divisor cannot be zero.